src/perfrecord.cpp

```diff
@@ -144,6 +124,42 @@ void PerfRecord::record(const QStringList &perfOptions, const QString &outputPat
     m_perfRecordProcess->start(perfBinary, perfCommand);
 }
 
+void PerfRecord::record(const QStringList &perfOptions, const QString &outputPath, const QStringList &pids,
```

Comment:
when attaching, we don't need a working directory - it's meaningless as we don't launch any new process (except perf, but that doesn't need a working directory - we set it's output path after all)